workload.go

```diff
@@ -47,7 +47,11 @@ type WorkloadConfig struct {
 	Name string
 
 	// The database config used to create connection objects.
-	DatabaseConfig DatabaseConfig
+	BenchmarkConfig *BenchmarkConfig
```

Comment:
Changed this so that rate control settings from the command line are available to be applied as defaults in the absence of `RateControl` settings in the `WorkloadConfig`.